src/components/Tooltip/__snapshots__/Tooltip.test.tsx.snap

```diff
@@ -0,0 +1,100 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Tooltip renders correctly with custom placement 1`] = `
+<DocumentFragment>
+  <div
+    id="popup-container"
+  >
+    <span
+      class="mia-platform-tooltip-open"
+    >
+      Tooltip trigger
+    </span>
+    <div
+      class="mia-platform-tooltip mia-platform-zoom-big-fast-appear mia-platform-zoom-big-fast-appear-prepare mia-platform-zoom-big-fast mia-platform-tooltip-placement-rightBottom"
```

Comment:
I think it's okay to remove them. I don't believe they would change with every antd update, because they do not have hashed ids. They might change if antd changes the classnames associated to certain props (e.g. positioning)